components/camel-hystrix/src/main/java/org/apache/camel/component/hystrix/processor/HystrixProcessorCommand.java

```diff
@@ -117,6 +117,15 @@ protected Message run() throws Exception {
             copy.setException(e);
         }
 
+        // if hystrix execution timeout is enabled and fallback is enabled and a timeout occurs
+        // then a hystrix timer thread executes the fallback so we can stop run() execution
+        if(getProperties().executionTimeoutEnabled().get()
```

Comment:
Checkstyle issue, eg add space after if (run mvn -P sourcecheck)

Reply:
Updated as requested even though `mvn clean isntall -Psourcecheck` does not signal the problem for me :(

Reply:
That is weird as there should be a space after the if, eg
`if(getProperties()` should be `if (getProperties()`

Reply:
didn't notice it since the build is not failing, the error was present in mvn log though, so my bad I guess :)

Reply:
@valdar yeah the behavior changed at some point, if you want the build to fail, you have to run with `-Psourcecheck -Dcheckstyle.failOnViolation=true`